File: engine/model/src/main/scala/tools/packet/foreigneffect/ShowForeignMakerEffect.scala
package tools.packet.foreigneffect

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class ShowForeignMakerEffect(private var _characterId: Int, private var _success: Boolean) extends PacketInput {
  def characterId: Int = _characterId

  def success: Boolean = _success

  override def opcode(): SendOpcode = SendOpcode.SHOW_FOREIGN_EFFECT
}

File: engine/engine-model/src/main/java/tools/packet/foreigneffect/ShowForeignMakerEffect.java
package tools.packet.foreigneffect;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record ShowForeignMakerEffect(Integer characterId, Boolean success) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.SHOW_FOREIGN_EFFECT;
   }
}